src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Android/Airship.cs

```diff
@@ -29,12 +26,16 @@ private void Init()
             
             //Adding Inbox updated listener
             MessageCenterClass.Shared().Inbox.AddListener(this);
+
+            UAirship.Shared().DeepLinkListener = this;
```

Comment:
This is problematic since it's not a single listener. Now by including the .net framework it takes over the listener instead of only doing it if you use this functionality. This could break other apps.